spec/declaration.dd

```diff
@@ -398,6 +399,14 @@ t1.t v1;  // v1 is type int
 t2 v2;    // v2 is type int
 t3 v3;    // v3 is type int
 t4 v4;    // v4 is type int
+
```

Comment:
Perhaps out of scope for this PR, but I noticed that even though one can infer from the grammar that aliasing function and delegate pointer types is supported, there's no example of that and I think this paragraph may be a good place to add this. Ditto for function literals, which are in general a less known feature.

Reply:
Someone can make another PR, i'll open an issue later to get track of this

Reply:
https://issues.dlang.org/show_bug.cgi?id=19824